src/code.cloudfoundry.org/cf-tcp-router/config/config.go

```diff
@@ -60,6 +73,13 @@ func New(path string) (*Config, error) {
 	return c, nil
 }
 
+func (c *Config) FrontendTLSJobBasePath() string {
+	if bp := os.Getenv("FRONTEND_TLS_BASE_PATH"); bp != "" {
+		return bp
+	}
+	return "/var/vcap/jobs/tcp_router/config/keys/tcp-router/frontend"
```

Comment:
do we have to worry about proper permissions? Copying the sibling's (`backend`) permissions:

```
chown -R root:vcap /var/vcap/jobs/tcp_router/config/certs/tcp-router/frontend
chmod -R 640 	  /var/vcap/jobs/tcp_router/config/certs/tcp-router/frontend
chmod      750 	  /var/vcap/jobs/tcp_router/config/certs/tcp-router/frontend
```